qiskit_optimization/translators/ising.py

```diff
@@ -148,9 +173,24 @@ def from_ising(
         QiskitOptimizationError: if any Pauli term has an imaginary coefficient
         NotImplementedError: If the input operator is a ListOp
     """
-    if isinstance(qubit_op, PauliSumOp):
-        qubit_op = qubit_op.to_pauli_op()
+    # quantum_info
+    if isinstance(qubit_op, BaseOperator):
+        if not isinstance(qubit_op, SparsePauliOp):
+            qubit_op = SparsePauliOp(qubit_op)
 
+    # opflow
+    if isinstance(qubit_op, OperatorBase):
+        warn(
+            "The `qubit_op` argument can currently accept Opflow operators, but it has been "
+            "superseded by `SparsePauliOp`. "
+            "Opflow will be deprecated in a future release and subsequently removed after that.",
```

Comment:
I did not alter the text here as per he above suggestion. But if the suggestion seems better then perhaps this should be the same.

Reply:
Thanks. I include it. cc41551